main.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from dotenv import load_dotenv
+from fastapi import FastAPI
+
+from cities.routers import router as city_routers
+
+load_dotenv()
+
+WEATHER_API_KEY = os.getenv("WEATHER_API_KEY")
+
+WEATHER_API_URL = os.getenv("WEATHER_API_URL")
```

Comment:
Ensure that the `WEATHER_API_URL` environment variable is set in your environment. If it's not set, the application may fail to construct the correct API endpoint for weather data.